eudemo/eudemo_tests/ivc/test_divertor_silhouette.py

```diff
@@ -120,9 +120,9 @@ def test_baffle_start_and_end_points_and_target_intersects(self):
         inner_baffle, inner_target, _, outer_target, outer_baffle = designer.execute()
 
         assert inner_baffle is not None
-        assert inner_baffle.start_point()[0] == min(designer.x_limits)
+        assert min(designer.x_limits) == pytest.approx(inner_baffle.end_point()[0][0])
```

Comment:
is this not quite yoda condition-y here? dont you want 
my_thing == approximately(value)

not 
value == approximately(my_thing)